### PR TITLE
docs: remove cargo test workspace command

### PR DESCRIPTION
**Pull request by waynexia**
## What's Changed in this PR

*Describe the change in this PR*

Related to https://github.com/GreptimeTeam/greptimedb/issues/4324

## Checklist

- [ ] Please confirm that all corresponding versions of the documents have been revised.
- [ ] Please ensure that the content in `summary.yml` matches the current document structure when you changed the document structure.
- [ ] This change requires follow-up update in localized docs.

## Summary by CodeRabbit

- **Documentation**
  - Updated test documentation to recommend using `cargo nextest run` for running Rust unit tests, improving test execution speed.

### CODE REVIEW

File: docs/nightly/en/contributor-guide/tests/unit-test.md [hunk 1]
```diff
@@ -3,12 +3,12 @@
 ## Introduction
 
 Unit tests are embedded into the codebase, usually placed next to the logic being tested.
-They are written using Rust's `#[test]` attribute and can run with `cargo test --workspace`.
+They are written using Rust's `#[test]` attribute and can run with `cargo nextest run`.
 Since `GreptimeDB` orchestrates its components in the "workspace" manner, the tailing
 `--workspace` is necessary to run all the unit cases.
```
Comment (evenyag) [inline docs/nightly/en/contributor-guide/tests/unit-test.md:8]:
```suggestion
```

File: docs/nightly/zh/contributor-guide/tests/unit-test.md [hunk 2]
```diff
@@ -3,7 +3,7 @@
 ## Introduction
 
 Unit tests are embedded into the codebase, usually placed next to the logic being tested.
-They are written using Rust's `#[test]` attribute and can run with `cargo test --workspace`.
+They are written using Rust's `#[test]` attribute and can run with `cargo nextest run`.
 Since `GreptimeDB` orchestrates its components in the "workspace" manner, the tailing
 `--workspace` is necessary to run all the unit cases.
```
Comment (evenyag) [inline docs/nightly/zh/contributor-guide/tests/unit-test.md:8]:
```suggestion
```

File: docs/v0.8/en/contributor-guide/tests/unit-test.md [hunk 3]
```diff
@@ -3,7 +3,7 @@
 ## Introduction
 
 Unit tests are embedded into the codebase, usually placed next to the logic being tested.
-They are written using Rust's `#[test]` attribute and can run with `cargo test --workspace`.
+They are written using Rust's `#[test]` attribute and can run with `cargo nextest run`.
 Since `GreptimeDB` orchestrates its components in the "workspace" manner, the tailing
 `--workspace` is necessary to run all the unit cases.
```
Comment (evenyag) [inline docs/v0.8/en/contributor-guide/tests/unit-test.md:8]:
```suggestion
```

File: docs/v0.8/zh/contributor-guide/tests/unit-test.md [hunk 4]
```diff
@@ -3,12 +3,12 @@
 ## Introduction
 
 Unit tests are embedded into the codebase, usually placed next to the logic being tested.
-They are written using Rust's `#[test]` attribute and can run with `cargo test --workspace`.
+They are written using Rust's `#[test]` attribute and can run with `cargo nextest run`.
 Since `GreptimeDB` orchestrates its components in the "workspace" manner, the tailing
 `--workspace` is necessary to run all the unit cases.
```
Comment (evenyag) [inline docs/v0.8/zh/contributor-guide/tests/unit-test.md:8]:
```suggestion
```

**Review by evenyag — APPROVED** [review]
We should remove the docs that mention `--workspace`.

**Review by nicecui — APPROVED** [review]

**Pull request merged**